app/AndroidManifest.xml

```diff
@@ -49,6 +49,11 @@
             android:authorities="org.commcare.dalvik.case"
             android:enabled="true"
             android:readPermission="org.commcare.dalvik.provider.cases.read" />
+        <provider
+            android:name=".provider.FixtureDataContentProvider"
+            android:authorities="org.commcare.dalvik.fixture"
+            android:enabled="true"
+            android:readPermission="org.commcare.dalvik.provider.fixtures.readFixtures" />

```

Comment:
Hm, I'm wondering whether we should just re-use the case read permission, and just rename it to be "CommCare User Data". Asking people to declare a bunch of permissions seems painful.